public/javascripts/Gallery/src/filter/CardFilter.js

```diff
@@ -116,18 +116,22 @@ function CardFilter(uiCardFilter, labelTypeMenu, cityMenu, initialFilters) {
 
         // For each type of filter, check if it matches the default. If it doesn't, add to URL in a query param.
         if (status.currentLabelType !== 'Assorted') {
+            $('#clear-filters').show();
```

Comment:
Could you add a reference to `#clear-filters` in `Main.js` so that we don't have to use the jquery selector in a bunch of places in this file? It will go in the section that follows `sg.ui.cardFilter = {};`. Then we should be able to access the button through the `uiCardFilter` object!